### ISSUE TITLE
Columns are not scrollable

### ISSUE THREAD

**Issue by NearW** [MaibornWolff/retro#196]
**Describe the bug**
We changed the behaviour so that not the whole page is scrollable, but the columns itself are. This is not working anymore.

**To Reproduce**

1. Create multiple cards, so that some cards are clipping out of the screen
2. Try to scroll within a column

**Expected behavior**
Each columns should be scrollable separately.

**Comment by Nomandes** [comment]
A working example from our used library can be found here: 
https://react-beautiful-dnd.netlify.app/?path=/story/board--scrollable-columns

**Comment by NearW** [comment]
> A working example from our used library can be found here: https://react-beautiful-dnd.netlify.app/?path=/story/board--scrollable-columns

They simply set a fixed height on the div. so this is the same as you did with the vh. :(

**Comment by Nomandes** [comment]
Yes for the height i will need a different solution, but they have nested scrolling without warnings from the library
